libmisc/prefix_flag.c

```diff
@@ -229,6 +229,7 @@ extern struct passwd *prefix_getpwnam(const char* name)
 		return getpwnam(name);
 	}
 }
+#if HAVE_FGETPWENT_R
```

Comment:
The #if HAVE_FGETPWENT_R is also needed around the lib/prototypes.h definition of prefix_getpwnam_r.

Reply:
Done, thank you.